sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ImmutableStorageWithVersioningAsyncTests.java

```diff
@@ -645,8 +646,11 @@ private static Stream<Arguments> blobUploadSupplier() {
         return Stream.of(Arguments.of(1L), Arguments.of((Long) null));
     }
 
+    // Live mode does not support copy from URL with immutability policy due to public access
+    // Revisit this and remove public access once default credential is enabled
     @RequiredServiceVersion(clazz = BlobServiceVersion.class, min = "2020-10-02")
     @Test
+    @PlaybackOnly()
```

Comment:
This test does not need to be made playback only, change like 665 to vlwBlob.GenerateSasUri() or whatever the method is in Java to generate a read-only SAS the expires in 1 hour, and this test will pass again in live mode.